code/modules/shuttle/shuttle_manipulator.dm

```diff
@@ -0,0 +1,289 @@
+		Somewhere ages and ages hence:\n\
+		Two roads diverged in a wood, and I,\n\
+		I took the one less traveled by,\n\
+		And that has made all the difference."
+
+	icon = 'icons/obj/machines/shuttle_manipulator.dmi'
+	icon_state = "holograph_on"
+
+	var/current_tab = SHUTTLE_MANIPULATOR_STATUS
+	var/busy
+	// UI state variables
+	var/datum/map_template/shuttle/selected
+
+	var/obj/docking_port/mobile/existing_shuttle
+
+	var/obj/docking_port/mobile/preview_shuttle
+	var/datum/map_template/shuttle/preview_template
+	var/list/templates = list()
+	var/list/shuttle_data = list()
+
+/obj/machinery/shuttle_manipulator/New()
+	. = ..()
+	update_icon()
+
+/obj/machinery/shuttle_manipulator/update_icon()
+	overlays.Cut()
+	var/image/hologram_projection = image(icon, "hologram_on")
+	hologram_projection.pixel_y = 22
+	var/image/hologram_ship = image(icon, "hologram_whiteship")
+	hologram_ship.pixel_y = 27
+	overlays += hologram_projection
+	overlays += hologram_ship
+
+/obj/machinery/shuttle_manipulator/process()
+	return
+
+/obj/machinery/shuttle_manipulator/attack_hand(mob/user as mob)
+	user.set_machine(src)
+	interact(user)
+
+/obj/machinery/shuttle_manipulator/interact(mob/user)
+
+	for(var/shuttle_id in shuttle_templates)
+		var/datum/map_template/shuttle/S = shuttle_templates[shuttle_id]
+
+		if(!templates[S.port_id])
+			templates[S.port_id] = list(
+				"port_id" = S.port_id,
+				"templates" = list())
+
+		var/list/L = list()
+		L["name"] = S.name
+		L["shuttle_id"] = S.shuttle_id
+		L["port_id"] = S.port_id
+		L["description"] = S.description
+		L["admin_notes"] = S.admin_notes
+
+		if(selected == S)
+			shuttle_data["selected"] = L
+
+		templates[S.port_id]["templates"] += list(L)
+
+
+		// Status panel
+	for(var/i in shuttle_master.mobile)
+		var/obj/docking_port/mobile/M = i
+		var/list/L = list()
+		L["name"] = M.name
+		L["id"] = M.id
+		L["timer"] = M.timer
+		L["timeleft"] = M.getTimerStr()
+		L["mode"] = capitalize(M.mode)
+		L["status"] = M.getStatusText()
+		if(M == existing_shuttle)
+			shuttle_data["existing_shuttle"] = L
+
+		shuttle_data["shuttles"] += list(L)
+
+	var/dat = ""
+	dat += "<center>"
+	dat += "<a href='?src=\ref[src];[src]=tab;tab=[SHUTTLE_MANIPULATOR_STATUS]' [current_tab == SHUTTLE_MANIPULATOR_STATUS ? "class='linkOn'" : ""]>Status</a>"
+	dat += "<a href='?src=\ref[src];[src]=tab;tab=[SHUTTLE_MANIPULATOR_TEMPLATE]' [current_tab == SHUTTLE_MANIPULATOR_TEMPLATE ? "class='linkOn'" : ""]>Templates</a>"
+	dat += "<a href='?src=\ref[src];[src]=tab;tab=[SHUTTLE_MANIPULATOR_MOD]' [current_tab == SHUTTLE_MANIPULATOR_MOD ? "class='linkOn'" : ""]>Modification</a>"
+	dat += "</center>"
+	dat += "<HR>"
+
+	if(busy)
+		dat += "<div class='statusDisplay'>Shuttle Manipulation in progress...</div>"
+	else
+		dat += "<div class='statusDisplay'>"
+
+		switch(current_tab)
+			if(SHUTTLE_MANIPULATOR_STATUS)
+				dat += "<div class='block'>"
+				for(var/list/shuttle in shuttle_data["shuttles"])
+					dat += "<span class='line'>"
```

Comment:
`span` is not able to affect the layout of the document - for that, you want `div`, which will use the `line` class properly, and split the lines as you want. You can also probably remove all those `br`'s now, too